omnij-rpc/src/integ/groovy/foundation/omni/BaseMainNetSpec.groovy

```diff
@@ -58,6 +58,6 @@ abstract class BaseMainNetSpec extends Specification implements OmniClientDelega
             mscVersion = info.mastercoreversion
         }
         log.info "Bitcoin version: ${info.version}"
-        log.info "Mastercore version: ${mscVersion}"
+        log.info "Omni Core version: ${mscVersion}"
```

Comment:
I think we're almost done with the rebrand in this codebase at this point. These are getting fewer and fewer.